tests/core/system-snap-refresh/task.yaml

```diff
@@ -62,7 +73,7 @@ execute: |
 
         # revert the target snap
         snap revert "$TARGET_SNAP_NAME" &> revert.log
-        MATCH "snapd is about to reboot the system" < revert.log
+        MATCH "waiting on external action to be completed" < revert.log
```

Comment:
because of this change, the test is failing
I think we need to add this part of the change with the original change, but not now in this cherry pick.